vllm/model_executor/models/mixtral.py

```diff
@@ -124,11 +126,14 @@ def __init__(
 
         if self.use_fp8:
             # WEIGHT_SCALE (for fp8)
-            self.w13_scale = nn.Parameter(torch.ones(self.num_total_experts,
-                                                     dtype=torch.float32),
+            # Allocate 2 scales for w1 and w3 respectively.
+            # They will be combined to a single scale after weight loading.
+            self.w13_scale = nn.Parameter(torch.zeros(self.num_total_experts,
```

Comment:
Why switch from ones to zeros? One is a valid scale but zero is not

Reply:
I changed this when debugging, but actually it shouldn't matter as this value will be overwritten anyways. When loading FP8 checkpoint, this value will be overwritten by the value in the checkpoint. When loading FP16 checkpoint, this value will be overwritten when quantizing FP16 weights to FP8 after loading. I can change it back to reduce confusion tho.